cardano-coin-selection.cabal

```diff
@@ -48,17 +71,16 @@ library
     , text
     , transformers
   hs-source-dirs:
-      src
+      src/library
```

Comment:
why not to name `lib` instead of `library` ?

Reply:
> why not to name lib instead of library ?

No good reason in particular.

Since everything else (`internal`, `test`, etc.) at this level was not an abbreviation, it seemed fine to use whole words at this level.

But we could change it to `lib` if people feel strongly about it. :+1: